package/doc/sphinx/source/documentation_pages/references.rst

```diff
@@ -36,26 +36,12 @@ description of the library. We will publish a complete self-contained
 paper with the upcoming 1.0 release of MDAnalysis, which will then
```

Comment:
... er, yes, well, ... I suppose we _really_ should

Reply:
One paper at the time please 🙈